api/room.go

```diff
@@ -77,6 +77,33 @@ func (r *Room) Handle(c *websocket.Conn, l *LoginReq) error {
 			}
 			r.mu.Unlock()
 			r.broadcast(marshalChatBatch(cm.ToItem()))
+		case "kick_user":
+			msg, err := unmarshalAs[KickMessage](raw)
+			if err != nil {
+				return fmt.Errorf("failed to unmarshal kick message: %s", err)
+			}
+			if msg.User == l.User {
+				r.broadcast(marshalChatBatch(ChatBatchItem{
```

Comment:
FYI - if you send the message this way, it will not be persisted in message history, and so will not be resent to people after they leave/join the room. Not sure if that's what we want :shrug: